modules/custom/dkan_harvest/dkan_harvest.api.php

```diff
@@ -14,11 +13,10 @@
  *   - override
  *   - def
  *   - excldue
- *   - datajsontodkan
+ *   - datajsontodkan.
```

Comment:
A lot of these small unrelated changes were made by PHPCBF trying to fix formatting